ethcore/src/blockchain/blockchain.rs

```diff
@@ -195,6 +195,77 @@ pub struct BlockChain {
 	pending_transaction_addresses: RwLock<HashMap<H256, Option<TransactionAddress>>>,
 }
 
+// TODO: Should this and `BestAncientBlock` be the same struct?
+#[derive(Clone, PartialEq, Eq)]
+struct Block {
```

Comment:
having this be called `Block` in the `blockchain` module is pretty confusing

Reply:
Agreed. Maybe `MinimalBlockMetadata` would be more accurate (if kinda verbose)